### PR TITLE
Support transfer tokens to multi-sig addresses

### PR DESCRIPTION
**Pull request by Lbqds**

### CODE REVIEW

File: ethereum/contracts/nft/NFTBridge.sol [hunk 1]
```diff
@@ -204,7 +204,7 @@ contract NFTBridge is NFTBridgeGovernance {
         require(bytes(transfer.uri).length <= 200, "tokenURI must not exceed 200 bytes");
 
         encoded = abi.encodePacked(
-            uint8(1),
+            uint8(3),
```
[the remark this answers is not in the document]
Comment (polarker) [reply ethereum/contracts/nft/NFTBridge.sol:207]: Are we going to deploy those NFT related contracts?

**Review by polarker — APPROVED** [review]
LGTM 👍

File: ethereum/contracts/nft/NFTBridge.sol (hunk 1 above)
[the remark this answers is not in the document]
Comment (polarker) [reply ethereum/contracts/nft/NFTBridge.sol:207]: IMO, we could keep the code, but don't need to deploy them for now. Is that possible?

[the remark this answers is not in the document]
Comment (Lbqds) [reply ethereum/contracts/nft/NFTBridge.sol:207]: Yes, I'll remove it from the deployment scripts.

File: alephium/contracts/constants.ral [hunk 2]
```diff
@@ -24,7 +24,6 @@ Abstract Contract Constants() {
     InvalidTransferTargetChain              = 19
     InvalidDestroyUnexecutedSequenceMessage = 20
     InvalidCaller                           = 21
-    InvalidTransferToAddress                = 22
```
Comment (polarker) [inline alephium/contracts/constants.ral:27]: May reduce the following error code by 1

Comment (Lbqds) [reply alephium/contracts/constants.ral:27]: Done

**Pull request merged**